charts/pelorus/Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: v1.2.1-rc-3-g4c285d1
+version: v1.2.1-3-g9a7b326 
```

Comment:
you're not touching the chart with this PR, so you shouldn't need a version bump.